lib/builders/gpu-builder.ts

```diff
@@ -0,0 +1,172 @@
+import { BlueprintBuilder } from '../../lib/stacks';
```

Comment:
this should be `../stacks`. Same applies to addons, utils, spi and clusterproviders